virt/monitoring/virt-running-cluster-checkups.adoc

```diff
@@ -22,28 +25,28 @@ include::snippets/technology-preview.adoc[]
 
 include::modules/virt-about-cluster-checkup-framework.adoc[leveloffset=+1]
 
```

Comment:
There needs to be some sort of ID above this heading, here's one option:
```suggestion
[id="virt-running-cluster-checkups-console_{context}"]
```

Reply:
Thanks!